cpp/src/slice2cs/CsUtil.h

```diff
@@ -6,88 +6,78 @@
 #include "../Ice/OutputUtil.h"
 #include "../Slice/Parser.h"
 
-namespace Slice
+namespace Slice::Csharp
```

Comment:
Unfortunately, the git diff is struggling here.
All I did was add `[[nodiscard]]`s, remove `static`, and added parameter names where they were missing.
Other, the function signatures and doc-comments are exactly the same.